File: backend/src/main/java/com/proj2/proj2/request/EventCreateRequest.java
package com.proj2.proj2.request;

import lombok.Data;

@Data
public class EventCreateRequest {
    
    private int dueYear;
    private int dueMonth;
    private int dueDay;
    private int dueHour;
    private int dueMin;
    private String text;
    private String title;
    private long adminid;
    private boolean isMonthly;

}

File: frontend/src/app/request/event-create-request.ts
export class EventCreateRequest
{
    dueYear: number = new Date().getFullYear();
    dueMonth: number = new Date().getMonth();
    dueDay: number = new Date().getUTCDate();
    dueHour: number = new Date().getHours();
    dueMin: number = new Date().getMinutes();
    text: string = "";
    title: string = "";
    adminid!: number;
    monthly: boolean = false;
}